compiler-core/src/type_/lsp_pretty.rs

```diff
@@ -0,0 +1,93 @@
+use std::sync::Arc;
+
+use ecow::EcoString;
+use im::HashMap;
+use itertools::Itertools;
+
+
+use crate::ast::TypeAst;
+
+use super::{pretty::Printer, Type, TypeVar};
```

Comment:
We can't use the type pretty printer as it adds linebreaks and indentation, we need to have a language server type printer that appends to a string

Reply:
Gotcha. Will change this.